.development.env

```diff
@@ -16,3 +16,6 @@ EQ_REDIS_PORT=6379
 EQ_SECRETS_FILE=dev-secrets.yml
 EQ_KEYS_FILE=dev-keys.yml
 AWS_DEFAULT_REGION=eu-west-1
+GUNICORN_WORKERS=1
+GUNICORN_KEEP_ALIVE=2
+GUNICORN_CMD_ARGS=-c gunicorn_config.py
```

Comment:
I think as this contains a space it may need to be wrapped in quotes to make sure nothing odd happens. In the example in the docs it's quoted - https://docs.gunicorn.org/en/stable/settings.html#settings.

Reply:
I've attempted adding the quotes and docker fails with them.

Reply:
As an env var is parsed as a string the current implementation of gunicorn should parse it correctly https://github.com/benoitc/gunicorn/blob/7d8c92f48a6d9cee6b15fbdade6981721182d073/gunicorn/config.py#L71

Reply:
Gunicorn does with or without, but we also specify the var as part of the docker setup which doesn't like the quotes.